hmpps-prisoner-search/src/main/resources/application.yml

```diff
@@ -20,7 +20,19 @@ spring:
     oauth2:
       resourceserver:
         jwt:
-          jwk-set-uri: ${api.base.url.oauth}/.well-known/jwks.json
+          jwk-set-uri: ${hmpps-auth.base-uri}/.well-known/jwks.json
+
+      client:
+        registration:
+          prison-api:
+            provider: hmpps-auth
+            client-id: ${prison-api.client.id}
+            client-secret: ${prison-api.client.secret}
+            authorization-grant-type: client_credentials
+            scope: read
+        provider:
+          hmpps-auth:
+            token-uri: ${api.base.url.oauth}/oauth/token
```

Comment:
shouldn't token-uri be the new hmpps-auth.base-uri etc

Reply:
🤦 - I thought I'd fixed all the references